packages/dds/tree/src/codec/codec.ts

```diff
@@ -53,10 +53,10 @@ export interface SchemaValidationFunction<Schema extends TSchema> {
  * However, persisted data can sometimes be corrupted, bugs can produce invalid data, or users can mix up which data is compatible with which APIs.
  * In such cases, a format validator can help catch issues.
  *
- * Current options are {@link FormatValidatorNoOp} and {@link FormatValidatorBasic}.
+ * Current options are {@link https://github.com/microsoft/FluidFramework/blob/main/packages/dds/tree/src/codec/codec.ts | FormatValidatorNoOp} and {@link https://github.com/microsoft/FluidFramework/blob/main/packages/dds/tree/src/external-utilities/typeboxValidator.ts | FormatValidatorBasic}.
```

Comment:
These two seem unnecessary, the types are exported so this feels intended for IntelliSense, not to point to the public docs.